Modules/Core/Common/include/itkGaussianDerivativeOperator.h

```diff
@@ -210,8 +210,6 @@ class ITK_TEMPLATE_EXPORT GaussianDerivativeOperator : public NeighborhoodOperat
   PrintSelf(std::ostream & os, Indent i) const override;
 
 protected:
-  using CoefficientVector = typename Superclass::CoefficientVector;
```

Comment:
Keeping these is probably needed, as per https://stackoverflow.com/questions/1643035/propagating-typedef-from-based-to-derived-class-for-template

Reply:
> Keeping these is probably needed

If so, wouldn't a simple C++98 `using` just be fine? As in:

    using typename Superclass::CoefficientVector;

Reply:
Yes, `using typename Superclass::CoefficientVector;` is sufficient and preferred.